buildpack/layermetadata.go

```diff
@@ -29,7 +27,7 @@ func EncodeLayerMetadataFile(lmf LayerMetadataFile, path, buildpackAPI string) e
 	encoders := supportedEncoderDecoders()
 
 	for _, encoder := range encoders {
-		if encoder.IsSupported(buildpackAPI) {
```

Comment:
We should bring the arg back so that this structure is useful if we ever decide to add another encoder